bin/fuel-convert

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env python
+import argparse
+import importlib
+import pkgutil
+
+from fuel import converters
+
+
+if __name__ == "__main__":
+    dataset_options = [
+        m[1] for m in pkgutil.iter_modules(converters.__path__) if not m[2]]
+    parser = argparse.ArgumentParser()
+    parser.add_argument("dataset", help="what dataset to convert",
+                        choices=dataset_options)
+    parser.add_argument("-d", "--directory", help="directory in which input " +
```

Comment:
I disagree on that; I think putting the HDF5 file where the dataset class expects it is a good default.